ZelBack/src/services/fluxCommunication.js

```diff
@@ -1103,7 +1116,7 @@ async function checkMyFluxAvailability(retryNumber = 0) {
   }
   let availabilityError = null;
   const axiosConfig = {
-    timeout: 7000,
+    timeout: 11000,
```

Comment:
Good I see it reflected here, shall be sufficient

Reply:
Maybe we shall do it in 2 step release, as if this check is not applied it may be an issue. However not getting a response of the 2 requests within 7 seconds is highly unlikely anyway